components/automate-ui/src/app/types/types.ts

```diff
@@ -64,6 +64,7 @@ export class ComplianceNode implements ComplianceData {
 export interface Chicklet {
   text: string;
   type: string;
+  type_key?: any;
```

Comment:
```suggestion
  type_key?: string;
```

I think all the type_keys are always strings, from what i've noticed